docs/howtos/DataPlane Generation - DPG/03convenient.mdx

```diff
@@ -168,9 +167,9 @@ The two possible value for the `Access` enum are `internal` and `public`.
 <TabItem value="python" label="Python">
 
 ```python
-class _GetModel:
-  ...
-# FIXME
+# can't import form models directly
+from petstorenamespace.models import GetModel # will report error
+from petstorenamespace.models._models import GetModel # OK but not recommended in Python
```

Comment:
we shouldn't document private imports

Reply:
I think we should just not even include the private imports

Reply:
Updated @iscai-msft